docs/howtos/DataPlane Generation - DPG/10longRunningOperations.mdx

```diff
@@ -0,0 +1,190 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Long-Running Operations
+
+This doc details what emitters will generate for long-running operations.
+
+## Resource create or replace
+
+Long-running operation to create or replace a resource.
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+alias ResourceOperations = global.Azure.Core.ResourceOperations<NoConditionalRequests &
+  NoRepeatableRequests &
+  NoClientRequestId>;
+
+@resource("users")
+@doc("Details about a user.")
+model User {
+  @key
+  @visibility("read")
+  @doc("The name of user.")
+  name: string;
```

Comment:
As the name is defined as read-only, why people can set its value through the SDK?

Reply:
In request, the name is on path (not body).

Reply:
@weidongxu-microsoft should we set it as readonly, or only as a key

Reply:
discussion with Mary here https://github.com/Azure/typespec-azure/pull/1361#discussion_r1718025420

Currently I am using the resource definition pattern in typespec-azure get-started doc
https://azure.github.io/typespec-azure/docs/getstarted/azure-core/step04

The alternative is to remove the `@visibility` and add a `@path`. Though this use may trigger a warning (probably depends on the template), e.g. 
```
path property will be ignored as it is inside of a @body property. Use @bodyRoot instead if wanting to mix.TypeSpec(@typespec/http/metadata-ignored)
```

The `@path` may also be involved a consistency discussion, as I remember currently we filter the `@path` (query/header too) from model/class.